internal/cri/config/config.go

```diff
@@ -427,6 +427,11 @@ type RuntimeConfig struct {
 	// IgnoreDeprecationWarnings is the list of the deprecation IDs (such as "io.containerd.deprecation/pull-schema-1-image")
 	// that should be ignored for checking "ContainerdHasNoDeprecationWarnings" condition.
 	IgnoreDeprecationWarnings []string `toml:"ignore_deprecation_warnings" json:"ignoreDeprecationWarnings"`
+
+	// UseSecureCapabilities indicates to use a more secure set of default capabilities
+	// when creating containers. This removes capabilities that are not typically
+	// needed for most workloads, such as NET_RAW, SETFCAP and AUDIT_WRITE.
+	UseSecureCapabilities bool `toml:"use_secure_capabilities" json:"useSecureCapabilities"`
```

Comment:
- "Secure" might be confusing. "Reduced" might be more accurate?
- Not sure a single boolean is enough in the long-term. Maybe this should be a string like `capability_profile = "compat" | "reduced"` ?

Alternatively maybe we can just drop NET_RAW by default and call it a day, as NET_RAW seems the most concerned cap.
That will be a breaking change, but not sure if it is worth bumping up the major version to v3.

Reply:
@AkihiroSuda  @mikebrow if we can decide on a path forward, I can add some tests and move this to review